cheroot/server.py

```diff
@@ -1553,9 +1206,9 @@ class HTTPServer:
     Default is 10. Set to None to have unlimited connections."""
 
     def __init__(
-        self, bind_addr, gateway,
-        minthreads=10, maxthreads=-1, server_name=None,
-        peercreds_enabled=False, peercreds_resolve_enabled=False,
+            self, bind_addr, gateway,
+            minthreads=1, maxthreads=1, server_name=None,
```

Comment:
The signature change doesn't look related to this PR? (As well as indentation)

Reply:
It absolutely isn't. I was having problems with the debugger getting stuck due to threading issues and had to tune the settings. I'll pull this out right away